frontend/metrics_program.html

```diff
@@ -1,26 +1,35 @@
+<!-- Generic -->
 <option value="ntrips_mode_confirm" data-sizex="4" data-sizey="4">Number of trips</option>
 <option value="ntrips_commute_mode_confirm" data-sizex="4" data-sizey="4">Number of commute trips</option>
 <option value="ntrips_purpose" data-sizex="4" data-sizey="4">Trip count by purpose</option>
 <option value="ntrips_under10miles_mode_confirm" data-sizex="4" data-sizey="4">Trip count under 10 miles</option>
 <option value="miles_mode_confirm" data-sizex="4" data-sizey="4">Trip miles by mode</option>
 <option value="average_miles_mode_confirm" data-sizex="6" data-sizey="4">Average trip length</option>
+<option value="average_miles_mode_confirm2" data-sizex="6" data-sizey="4">Average trip length (selected by users)</option>
 <option value="ntrips_per_day" data-sizex="6" data-sizey="4">Trip frequency</option>
+<option value="ntrips_per_weekday" data-sizex="6" data-sizey="4">Trip frequency (weekday)</option>
+<option value="ts_emissions_user" data-sizex="8" data-sizey="2">Timeseries of emissions</option>
+<option value="ts_energy_user" data-sizex="8" data-sizey="2">Timeseries of energy</option>
+<option value="ts_emissions_vmt" data-sizex="8" data-sizey="2">Timeseries of emissions per mile</option>
+<option value="ts_users" data-sizex="8" data-sizey="2">Timeseries of active users</option>
+<option value="ts_all_modes" data-sizex="8" data-sizey="2">Timeseries of all mode shares</option>
+<!-- Mode Specific -->
+<option value="ntrips_replaced_mode" data-sizex="4" data-sizey="4">Trip miles by replaced mode</option>
 <option value="ntrips_ebike_purpose" data-sizex="4" data-sizey="4">eBike trip count by purpose</option>
-<option value="ntrips_ebike_replaced_mode" data-sizex="4" data-sizey="4">eBike trip count by replaced mode</option>
```

Comment:
Renamed all hard-coded plot names

Reply:
You renamed all of them to "mode specific", which doesn't mean that much to an end user since it is not clear which mode you are targeting.

I think we should fill this in with the mode of interest instead. There are jquery templating plugins
https://www.endyourif.com/jquery-creating-templates-for-your-html-content/#jquery-template
but we should also be able to do something simpler with string replacement in javascript